curriculum/challenges/english/blocks/lecture-introduction-to-python/67fe859a00971c34a23abd43.md

```diff
@@ -25,6 +27,18 @@ int age = 25
 
 The dynamic-typing nature of Python makes coding really fast and more flexible, but it can lead to unexpected bugs because type errors are detected only when a program runs, not when the program compiles.
 
+Since Python determines data types while your program is **running**, type-related mistakes are only discovered at that moment. When a program runs, Python executes your code line by line. If it reaches a line where a type mistake happens, it will stop and show an error.
```

Comment:
```suggestion
Since Python determines data types while your program is **running**, type-related mistakes are only discovered at that moment. When a program runs, Python executes your code line by line. If it reaches a line where a certain object is expected to behave in a way it's not able to, Python will stop and show an error.
```
I'm also not sure about the _determines_ word. Unless there's explicit type check in code, there's no determination what type object has. In such case errors come from ie. not having specific method, that's required to do something.